bin/run-web-tool

```diff
@@ -1,3 +1,3 @@
 #! /bin/bash
 
-$(pwd)/bin/start-containter-with "/bound/bin/run-r-scripts"
+"$(pwd)/bin/start-containter-with" /bound/bin/run-r-scripts
```

Comment:
Pretty sure this should be
`"$(pwd)"/bin/start-containter-with /bound/bin/run-r-scripts`
or if you want to prepare for the future possibility of changing the sub-directory path to something else that also might have a space in it
`""$(pwd)"/bin/start-containter-with" /bound/bin/run-r-scripts`